skyplane/api/impl/transfer_job.py

```diff
@@ -152,6 +187,7 @@ class CopyJob(TransferJob):
     multipart_transfer_list: list = field(default_factory=list)
 
     def __post_init__(self):
+        self.type = "copy"
```

Comment:
Add a "type" class attribute to the dataclass with a default value. That way, it will serialize cleanly.